lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -118,6 +120,17 @@ class LighthouseReportViewer {
         this._reportIsFromGist = true;
         this._replaceReportHtml(reportJson);
       }).catch(err => logger.error(err.message));
+    } else if (jsonurl) {
+      const firebaseAuth = this._github.getFirebaseAuth(); // new method that needs to be added to return `this._auth`
```

Comment:
the comment here was a message to you that `GithubApi` class will need to be updated to add the new method that returns `this._auth`, not meant to be in the code :)

Reply:
hmm lol, very bad way to suggest a change

Reply:
The suggestion was just trying to help show what we needed to do, not verbatim all the changes that were necessary. That's what the "I think it'll end up looking something like" was meant to convey, but I certainly could have made it clearer :)

You'll need to change the github API class to add this new method `getFirebaseAuth` that returns `this._auth` from the github API class. AFAIK that should get us working again, but again, that was not fully tested and working guide, just meant as a guideline, so you'll want to run through the flow to make sure it works.

Reply:
👍